lib/generators/draper/install_generator.rb

```diff
@@ -1,7 +1,7 @@
 module Draper
   module Generators
     class InstallGenerator < Rails::Generators::Base
-      source_root File.expand_path('../templates', __FILE__)
+      source_root File.expand_path("templates", __dir__)
```

Comment:
I agree that it is easier to understand conceptually. My only concern is that it differs from the Rails convention. I'd be curious what the Rails core team would say to a pull request to change this in the Rails codebase. Either way, I am fine with. Thanks for the changes!